### DISCUSSION TITLE
Add open-tracking

### DISCUSSION THREAD

**Discussion by alemswaj** [tutao/tutanota#3049] [Ideas]
Hi,
Can you add an open tracking system to  Tutanota ?
I think it's a very useful option that I would like to see in Tutanota.

**Comment by jowlo** [comment]
Hi!
What do you mean by "open tracking system"? Tracking GPS locations/tracks/sports?
Could you describe your idea or feature in more detail?

**Reply by alemswaj** [reply]
No I mean a tacking system to know when the receiver has open the mail

**Reply by jowlo** [reply]
Aaah, now i understand!
We might discuss this further but imho we will never add such a functionality. One of our explicit goals is to enable our users to **not** be susceptible to this kind of tracking when done by others. It would be quite strange if we implemented it ourselves.

**Reply by alemswaj** [reply]
Ok so nobody can know if a user of Tutanota had open a mail ?
Well, if they can't know if we had open a mail, it's normal that we can't know either.
Thanks for your answer.
And just another question, how do you prevent others from tracking us ?
I don't know if it's clear, I don't speak English.

**Reply by jowlo** [reply]
Well, there are multiple ways that senders (mostly marketing newsletters) try to track whether a mail was opened, you can read a bit more about it [here on wikipedia](https://en.wikipedia.org/wiki/Email_tracking), or do a quick web search which gives many results.

We block all external images, remove any javascript, mark links that actually lead to something else than what they appear to be as phishing, and more to not give away whether our users opened a mail. So, basically, everything that would be loaded from somewhere else on the web could reveal that you are accessing that resource because you opened the mail, and we block it. Also, we do not answer to automated read receipt mails.

This may not always be perfect, for example if you still click a link in an e-mail or click on the icon to load external images you might still reveal to the sender that you have read the mail.
